open-trip-planner/src/main/scala/com/azavea/busplan/routing/StudentToStopRouting.scala

```diff
@@ -34,12 +36,13 @@ object StudentToStopRouting {
   }
 
   def getMaxDistance(maxDistance: Double, grade: Int): Double = {
+    var newMaxDistance = maxDistance;
     if (grade < 7) {
-      if (maxDistance > 2640) {
-        val maxDistance = 2640
+      if (maxDistance >= 804) {
+        newMaxDistance = 804
```

Comment:
There's a number of magic numbers for distances - it would be good to add them to a `Constants` object, so we can refer to them as e.g. `Constants.QUARTER_MILE`.